components/Feature/StatusForm/index.js

```diff
@@ -1,10 +1,19 @@
 import { REFERRAL_STATUSES } from 'lib/utils/constants';
 import { useState } from 'react';
+import css from './index.module.css';
 
 const StatusForm = ({ onSubmitForm, name }) => {
   const [reject, setReject] = useState(false);
   const [status, setStatus] = useState();
   const [comment, setComment] = useState();
+  const [validationError, setValidationError] = useState({ rejectReason: false });
+
+  const onChangeRejectReason = event => {
+    const fieldValue = event.target.value;
+    const isValueEmpty = !Boolean(fieldValue);
+    setComment(fieldValue);
+    setValidationError({ ...validationError, rejectReason: isValueEmpty });
```

Comment:
rejectReason might be better as isRejectReasonEmpty or something that infers it's a boolean as this one looks like it stores the text for reject reason.